javascript/0540-single-element-in-a-sorted-array.js

```diff
@@ -0,0 +1,20 @@
+// Time Complexity: O(log n)
+// Space Complexity: O(1)
+
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var singleNonDuplicate = function (nums) {
+    let l = 0,
```

Comment:
Suggestion: name descriptive variables

Reply:
Updated the code. Can you check this PR as well

https://github.com/neetcode-gh/leetcode/pull/2692

Thanks